### DISCUSSION TITLE
Frage zu den ersten 3 Einträgen

### DISCUSSION THREAD

**Discussion by ev4x** [haumacher/phoneblock#25] [Q&A] [answered]
SPAM: 0
SPAM: 00
SPAM: 0000

Warum sind diese Einträge drin die Nummern gibt es doch garnicht und wenn sie 0 Sperren würde wäre jedes Ortsgerspräch gesperrt das mit 0 beginnt bei 00 jedes Telefonat aus dem Ausland oder wird es nur gesperrt wenn die Nummer exakt übereinstimmt? Wenn man das Ausland komplett sperren wollte wie wäre da die Syntax 00* ?

**Comment by haumacher** [comment, marked as answer]
Es gibt immer wieder Spaßvögel, die solche Nummern-Präfixe als Spam berichten. Die Nummern in der Blocklist sind aber nur exakte Nummern, daher haben diese Einträge keinen Effekt. Wenn man `00*` in der Blockliste hätte, wären de-facto alle Auslandsnummern gesperrt. Aus gutem Grunde lässt [PhoneBlock](https://phoneblock.haumacher.de/) die Eintragungen von solchen Wildcards aber nicht zu - sonst könnte man mit `0*` ganz einfach alle Anrufe sperren.

Wenn man für sich selber alle Auslandsnummer sperren will, dann kann man das gerne in einer eigenen persönlichen Blocklist in der Fritz!Box machen. 

Ich war schonmal dabei einen Algorithmus zu implementieren, der solche Unsinnseinträge wie `00` automatisch löscht (weil `00` ein Präfix von anderen Nummern in der Liste ist und daher selber keine Nummer sein kann). Allerdings habe ich bisher noch keine Zeit gefunden, das wirklich fertig zu machen.
